challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
```

Comment:
Why is this a global variable? It should be a local variable in the function.

No point in having a global variable here. It should be a local variable in the function.

Global variables should be avoided as much as possible as they can lead to bugs and make the code harder to maintain.

It is better to have local variables in functions as they are only accessible within the function and not outside of it. This makes the code more modular and easier to maintain.

Also, the variable name is not very descriptive.

When using global variables it should be uppercase and snake case. e.g. CARD_NUMBER.